.travis.yml

```diff
@@ -1,30 +1,48 @@
 language: python
-
-python:
-  - 2.6
-  - 2.7
-  - 3.3
-  - 3.4
-  - 3.5
-  - 3.6
-  - pypy
-  - pypy3
-
-install:
-    - python -c "import sys; sys.exit(sys.version_info[:2] != (3,2))" 2>/dev/null || pip install -U pip wheel
-    - pip install --install-option="--no-cython-compile" -r requirements.txt
-    - pip install -U beautifulsoup4 cssselect
-
-script:
-  - python -u setup.py clean
-  - CFLAGS="-O0 -g" python -u setup.py build_ext --inplace
-  - CFLAGS="-O0 -g" PYTHONUNBUFFERED=x make test
-
 matrix:
   allow_failures:
-    - python: pypy
-    - python: pypy3
-
+  - env: PYENV_VERSION=3.6.1
+  - python: pypy
+  - python: pypy3
+  include:
+  - os: linux
+    python: 2.6
+  - os: linux
+    python: 2.7
+  - os: linux
+    python: 3.3
+  - os: linux
+    python: 3.4
+  - os: linux
+    python: 3.5
+  - os: linux
+    python: 3.6
+  - os: linux
+    python: pypy
+  - os: linux
+    python: pypy3
+  - os: osx
+    language: generic
+    env: PYENV_VERSION=2.6.9
+  - os: osx
+    language: generic
+    env: PYENV_VERSION=2.7.13
+  - os: osx
+    language: generic
+    env: PYENV_VERSION=3.6.1
+install:
+- "travis/travisInstall.sh"
+script:
+- "travis/travisRun.sh"
 cache:
   directories:
-    - $HOME/.cache/pip
+  - "$HOME/.cache/pip"
+deploy:
```

Comment:
This section would have to be changed in master. It currently sends stuff to my repo which obviously wont work if merged. 

Basically you update the encrypted api_key to yours (travis can encrypts with the command line client)

If you are feeling brave you can also have travis push to pypi